instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -158,13 +158,15 @@ def response_hook(span: Span, environ: WSGIEnvironment, status: str, response_he
 import functools
 import typing
 import wsgiref.util as wsgiref_util
+from timeit import default_timer
 
 from opentelemetry import context, trace
 from opentelemetry.instrumentation.utils import (
     _start_internal_or_server_span,
     http_status_to_status_code,
 )
```

Comment:
Can you add example usage of these metrics in the doc strings (maybe in flask or django)?

Reply:
This is not an opt-in feature. The existing example already cover it. What do you want to see in the doc string?

Reply:
Similar to [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1110/files)

Reply:
We didn't want to include any SDK setup in the instrumentation docs. That's why all of the instrumentation inline docs for tracing show how to setup just the instrumentation.

Reply:
Right. We should have some way that this should be exposed to the user though. Maybe in examples or getting started?

Reply:
>We should have some way that this should be exposed to the user though

That the instrumentation supports metrics? I am not sure if it's worth adding example each time we add metric instrumentation and it's no different from existing example. What I think would be helpful is if there is table which shows if instrumentation has metrics support yet.

Reply:
I like this idea. Do you want to create one as part of this PR or create an issue for it?

Reply:
I can add that.

Reply:
Done here https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1128/files#diff-50998f4ff4326f7db47af0fe368ec0a8c2ddbae864cb40384355766c564910d6